### ISSUE TITLE
Task.Run() creates too much garbage

### ISSUE THREAD

**Issue by ZackOfAllTrades** [CesiumGS/cesium-unity#204]
![Task Run](https://user-images.githubusercontent.com/13579475/217312941-cab7f6f2-94ca-4a8b-bbc5-683c2e8ef843.PNG)

In this screen shot you can see that calling System::Threading::Tasks::Task::Run(f); inside the UnityTaskProcessor::startTask function is creating too much garbage.

I switched to std::thread(f).detach() and see no problems.

**Comment by ZackOfAllTrades** [comment]
I don't think Unity's WebRequest is that great either, 
![image](https://user-images.githubusercontent.com/13579475/217593543-4d4594e9-f36c-4af9-9766-b6f1231adf89.png)

Is there a reason to use it over cpr or winhttp.h?
